### ISSUE TITLE
'unarchiveObjectWithData:' is deprecated: first deprecated in iOS 12.0 - Use +unarchivedObjectOfClass:fromData:error: instead

### ISSUE THREAD

**Issue by andesappal** [pikaju/flutter-braintree#77]
https://github.com/braintree/braintree_ios/issues/718

**Comment by andesappal** [comment]
![image](https://user-images.githubusercontent.com/54340176/127376528-08057739-003d-49c3-b69f-90734f5f11aa.png)
@pikaju changed my podspec isn't changing Podilfe.lock versions. Could you make the update on your repo?

**Comment by andesappal** [comment]
We need to make these changes:
```
s.dependency 'Braintree/PayPal', '~> 5.0.0'
s.dependency 'Braintree/Apple-Pay', '~> 5.0.0'
```

**Comment by andesappal** [comment]
Hi @pikaju here's a PR https://github.com/pikaju/flutter-braintree/pull/78, thank you

**Comment by pikaju** [comment]
Thank you @andesappal, I published version `2.0.0`, I hope that fixes your issue.
